extensions/arc/package.json

```diff
@@ -144,9 +144,8 @@
             "notebookWizard": {
               "notebook": "./notebooks/arcDeployment/deploy.arc.data.controller.ipynb",
               "type": "new-arc-control-plane",
-              "runNotebook": false,
+              "runNotebook": "userChooses",
```

Comment:
can you come up with a better name here? 😄

Reply:
FWIW I don't have a huge issue with the name as it currently is - but I do slightly prefer that we keep the type the same (so some other string values besides boolean true and false)

Reply:
it should an adjective here, right? how about 'userSelected'?

Reply:
I suck at naming. I would appreciate suggestions.

Reply:
This is probably fine for now - eventually we may want to allow a bit more flexibility (have more types of actions) but I'm not too worried about making this super generic right now.

I would suggest removing the enabled property and then just having the non-OK options be optional so if they aren't included they won't show up.

Reply:
Sounds good. So, then the doneAction is always defined so a user will always have the option of running the notebook and optionally the wizard can add a scriptAction to allow for just scripting of the notebook.